spring-cloud-gcp-samples/spring-cloud-gcp-data-jpa-sample/README.adoc

```diff
@@ -1,33 +1,38 @@
-= Spring Data JPA with Spring Cloud GCP SQL Code Sample
+= Spring Data JPA with Spring Cloud GCP SQL Sample
 
 This sample application demonstrates how to use
 https://projects.spring.io/spring-data-jpa/[Spring Data JPA] with the
 link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql[Spring Cloud GCP SQL Starter].
 
+We will create an instance, a database within the instance, populate the database and then query it.
+
 == Setup
 
-* If you don't have a Google Cloud Platform project with billing enabled, create one from the
-https://console.cloud.google.com/[Google Cloud Console].
+1. Follow https://cloud.google.com/sql/docs/mysql/quickstart[these instructions] to set up a Google
+Cloud Project with billing enabled.
 
-* To authenticate, you have two options:
+2. Create a Google Cloud SQL instance from the
+https://console.cloud.google.com/sql/instances[Google Cloud Console Cloud SQL page].
 
-1. Download the https://cloud.google.com/sdk/[Google Cloud SDK] and run the following command:
-+
-`$ gcloud auth application-default login`
+3. Still within the Google Cloud Console SQL page, create a new database in the instance from the
+"Databases" section. **We recommend you to setup a root password** for your instance footnoteref:[passwordless, if you are using MySQL, you can actually create the database instance without root password!]
```

Comment:
Sorry to be beating on a dead horse, but I would rather we didn't "recommend" people to create a root password. It should be safe enough to rely on the OAuth2 authentication and it makes config simpler to not have one there.
If the password is mandatory for PostgreSQL, how about we only add a parenthesis saying something like "PostgreSQL requires a password. Please set one and make note of it"?